spec/models/contestant_spec.rb

```diff
@@ -6,4 +6,27 @@
     it {should have_many :contestant_projects}
     it {should have_many(:projects).through(:contestant_projects)}
   end
+
+  describe "instance methods" do
+    describe "#project_names" do
+      it "returns a string of project names the contestant has worked on" do
+        recycled_material_challenge = Challenge.create!(theme: "Recycled Material", project_budget: 1000)
+        news_chic = recycled_material_challenge.projects.create!(name: "News Chic", material: "Newspaper")
+        jay = Contestant.create!(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+        ContestantProject.create!(contestant_id: jay.id, project_id: news_chic.id)
+
+        expect(jay.project_names).to eq("News Chic")
+      end
+    end
+  end
+  describe "class methods" do
+    describe "#average_years_of_experience" do
+      it "returns the average years of experience for all contestants" do
+        jay = Contestant.create!(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+        gretchen = Contestant.create!(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+
+        expect(Contestant.average_years_of_experience).to eq(12.5)
+      end
```

Comment:
This method isn't exactly right. We wanted the average years of experience for the contestants that worked on a specific project, not all the contestants in the db. 
The way you're calling this method in the view is also a bit off. You've written this as a class method, but you called it like this: `@project.contestants.average_years_of_experience`, but `@project.contestants` returns an Array of contestant objects so that wouldn't quite be a Class method on the Contestant class. It works because it's not actually returning an array, but an active record relation, but it's not quite correct. I'd rather have this be an instance method written in the Project class since we're talking about contestants for a given project.